Code/GraphMol/SmilesParse/SmilesWrite.cpp

```diff
@@ -422,9 +422,9 @@ std::string MolToSmiles(const ROMol &mol, bool doIsomericSmiles, bool doKekule,
     if (canonical) {
       if (tmol->hasProp("_canonicalRankingNumbers")) {
```

Comment:
prefer common_properties::_canonicalRankingNumber if possible, should be quicker in general.